### DISCUSSION TITLE
Pulling annotations when using Adobe Acrobat Reader for opening PDfs on Zotero.

### DISCUSSION THREAD

**Discussion by gayatriindukumar** [aidenlx/obsidian-zotero#401] [Help]
I have set up Adobe Acrobat as the default reader on Zotero and annotate my literature there. Can these annotations be pulled into the Obsidian Note? I am new to the plugin and still trying to figure it out. I followed the instructions in the tutorial (Right-click>Obsidian Actions>Create Literature Note(s)) but I only got a note with a link to the PDF (not very helpful). Let me know if I need to be doing something differently here.

**Comment by aidenlx** [comment]
v2 reads external annotations. Once Zotero has picked up annotations authored outside its own reader — Acrobat included — they live in Zotero's database, and ZotLit renders them alongside native ones rather than skipping them. That is why you were only getting a note with a PDF link before.

Each annotation exposes `zt.isExternal` and `zt.authorName`, so you can label or filter them in the annotation template if you want to tell them apart:

```liquid
{% if zt.isExternal %}(imported){% endif %}
```

Field reference: https://zotlit.aidenlx.site/docs/reference/templates/data

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
